### ISSUE TITLE
Icons in module content

### ISSUE THREAD

**Issue by bradmac** [Kademi/kademi-dev#999]
Thea has reported the problems below.

![image001](https://cloud.githubusercontent.com/assets/826458/11328250/8db19a8c-91c1-11e5-859e-cbd5d99fd60d.jpg)
![image002](https://cloud.githubusercontent.com/assets/826458/11328251/8dbd7bf4-91c1-11e5-8b2d-8711826761f7.jpg)
![image003](https://cloud.githubusercontent.com/assets/826458/11328252/8dc0ef82-91c1-11e5-9bc2-cb5c5919537c.jpg)

**Comment by ducdhm** [comment]
Done!

**Comment by ducdhm** [comment]
@bradmac They're using really old style and js file of us like jquery.form, jquery.comment, jquery.user, register.js... Should we suggestion them to using new version (stable version) and change icons from sprites image to bootstrap/fontawesome icons?

**Comment by bradmac** [comment]
This is something we need to do

They need to have same styles available in the editor

We cannot expect them to have to change their existing content

Sent with AquaMail for Android
http://www.aqua-mail.com

**Comment by bradmac** [comment]
Just to clarify, this is our client, we built the styles and we upgraded the site to bootstrap. So we need to make sure their content works.

The styles that 3DN uses are generic elearning styles and should be standard components in our platform, and integrated into the editor. 

Note that 3DN will not be using the content editor, because to do so would mean migrating their existing content, which is too big a task.
